nautobot/utilities/tables.py

```diff
@@ -317,3 +330,39 @@ def render(self, value):
             trunc = Truncator(value)
             value = trunc.words(self.truncate_words)
         return value
+
+
+class CustomFieldColumn(tables.Column):
+    """
+    Display custom fields in the appropriate format.
+    """
+
+    def __init__(self, customfield, *args, **kwargs):
+        self.customfield = customfield
+        kwargs["accessor"] = Accessor(f"custom_field_data__{customfield.name}")
+        kwargs["verbose_name"] = customfield.label or customfield.name
+
+        super().__init__(*args, **kwargs)
+
+    def render(self, record, bound_column, value):
+        if value is None:
+            return self.default
+
+        if self.customfield.type == CustomFieldTypeChoices.TYPE_BOOLEAN:
+            if value is True:
```

Comment:
Maybe just `if value:` so that 'truthy' values don't get rendered as if they were `False`?

Reply:
I don't think TYPE_BOOLEAN can be anything but a boolean?

Reply:
Fair enough. I may just being paranoid here. :-)